docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   information are extended or withheld. Examples include varying accuracy in
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This is intended to illuminate the concept to readers who are trying to build AI systems in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
+
+Key Terms 
+~~~~~~~~~
+
+- **Sociotechnical context** – The context surrounding a technical system, including both social aspects (e.g., people, institutions, communities) and technical aspects (e.g., algorithms, technical processes). The sociotechnical context of a system shapes who might benefit or is harmed by AI systems.
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
+
+- **Measurement model** – The method and approach used to measure the unobservable theoretical construct.
```

Comment:
It may be good to add "via inferences from observable properties" or something like that from the paper. Since that connection between the unobservable construct and the observable properties is what they'll be testing for later.

Reply:
Good point - "by mapping" maybe? But agreed that that's still a bit vague and yeah examples would help (although fwiw @michaelamoako, future PRs can keep adding to this!)